interceptors/LoadEnvForCommands.cfc

```diff
@@ -1,24 +1,29 @@
 component {
-
-    property name="envFileName" inject="commandbox:moduleSettings:commandbox-dotenv:fileName";
-    property name="printOnLoad" inject="commandbox:moduleSettings:commandbox-dotenv:printOnLoad";
+    property name="moduleSettings" inject="commandbox:moduleSettings:commandbox-dotenv";
     property name="envFileService" inject="EnvironmentFileService@commandbox-dotenv";
-    property name="javaSystem" inject="java:java.lang.System";
     property name="fileSystemUtil" inject="FileSystem";
     property name='consoleLogger' inject='logbox:logger:console';
 
     function onCLIStart( interceptData ) {
         var directory = fileSystemUtil.resolvePath( "" );
-        var envStruct = envFileService.getEnvStruct( "#directory#/#envFileName#" );
-        if( !structIsEmpty( envStruct ) ) {
-            consoleLogger.info( "commandbox-dotenv: Loading Java properties from #directory##envFileName#" );
+        var envStruct = envFileService.getEnvStruct( "#directory#/#moduleSettings.fileName#" );
+        if( !structIsEmpty( envStruct ) && moduleSettings.printOnLoad ) {
```

Comment:
I like adding the option to completely silence the output from this module.  Originally it always printed out that it had loaded but would only print out each environment variable when `printOnLoad` was set.  Perhaps we should add a `verbose` setting as well.  That setting would be passed in to the `loadEnvToCLI` function.

Reply:
Sure, we can do two levels of output.  I just made it so I could turn everything off since running before very command just became intolerable after about 1 minute :)  It was handy to see when I was debugging, but once I knew it all worked, I was fine to have it all just go away.  I think the possible debugging messages currently fall into these two categories:

* Hey, I found a file to read!
* Hey, I loaded a specific var out of that file!